tasks/libs/copyright.py

```diff
@@ -36,6 +36,7 @@
     '/pkg/security/secl/model/.*_gen(_test){,1}.go',
     '/pkg/security/secl/model/accessors.go',
     '/pkg/trace/pb/.*_gen(_test){,1}.go',
+    '/pkg/clusteragent/custommetrics/api/generated/',
```

Comment:
It looks like this got file updated anyway?

Reply:
Also, `pkg/security/probe/` and `pkg/process/events/model/model_sysprobe_gen.go` appear to contain some generated files as well.